sdk/storage/azure-storage-blob/azure/storage/blob/_shared_access_signature.py

```diff
@@ -467,6 +474,7 @@ def generate_blob_sas(
         container_name,  # type: str
         blob_name,  # type: str
         snapshot=None,  # type: Optional[str]
+        version_id=None,  # type: Optional[str]
```

Comment:
Same comment as above - needs to move to the end of the parameters, or it could be a breaking change.